iCCN_for_PNG.md

```diff
@@ -0,0 +1,26 @@
+# iCCN
+
+## iCCN Embedded ICC profile (updated)
+The four-byte chunk type field contains the decimal values
+
+`105 67 67 80`
+
+The iCCN chunk contains:
+
+| Profile name |  1-79 bytes (character string)
+| Null separator |	1 byte (null character)
+| Compression method |	1 byte
+| Compressed profile |	n bytes
+
+The profile name may be any convenient name for referring to the profile. It is case-sensitive. Profile names shall be encoded as UTF-8. Leading, trailing, and consecutive spaces are not permitted. The only compression method defined in this International Standard is method 0 (zlib datastream with deflate compression, see 10.3: Other uses of compression). The compression method entry is followed by a compressed datastream of an ICC profile as defined in [ICC] or [ICC-2001]. The ICC profile shall either be an output profile (Device Class = `prtr`) or a monitor profile (Device Class = `mntr`). Decompression of this datastream yields the embedded ICC profile.
```

Comment:
I would like to understand the restriction to Output or Display profiles.

For example, [CSS `@color-profile`](https://drafts.csswg.org/css-color-4/#at-profile) says:

>  Input, Display, Output, or ColorSpace ICC profile. (Abstract, DeviceLink, and NamedColor ICC Profiles must not be used).

Reply:
Same rationale - to restrict the types of profiles that can be used and that make sense in this context.